R/vendoring.R

```diff
@@ -0,0 +1,55 @@
+#' Vendor dependent Rust crates
+#'
+#' Vendoring all dependent Rust crates into a single tarball.
+#' @inheritParams use_extendr
+#' @return No return value, called for side effects.
+#' @noRd
```

Comment:
Curious, why `@noRd`?

Reply:
> Curious, why `@noRd`?

As I said in the comments, I don't think it's worth running this function by itself.